.github/workflows/deploy_prod.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         uses: metcalfc/changelog-generator@v0.4.0
         with:
           myToken: ${{ secrets.GITHUB_TOKEN }}
-          base-ref: "prod-0" #need a tag "prod-0 for setup, please remove after setup"
+          # base-ref: "prod-0" #need a tag "prod-0 for setup, please remove after setup"
```

Comment:
Commented out as already have a reference from the latest release

Reply:
Let's delete this?